Sources/AWSLambdaRuntimeCore/LambdaRuntime.swift

```diff
@@ -19,34 +19,58 @@ import NIOCore
 /// `LambdaRuntime` manages the Lambda process lifecycle.
 ///
 /// Use this API, if you build a higher level web framework which shall be able to run inside the Lambda environment.
-public final class LambdaRuntime<Handler: ByteBufferLambdaHandler> {
+public final class LambdaRuntime {
```

Comment:
We did we drop the type here?

Reply:
this was required to remove the conformance to `ByteBufferLambdaHandler` from the other protocols, as otherwise it would require it. Do you have a better solution in mind?